miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/permission/MiniAppCustomPermissionCache.kt

```diff
@@ -96,44 +96,17 @@ internal class MiniAppCustomPermissionCache(context: Context) {
         applyStoringPermissions(MiniAppCustomPermission(miniAppId, allPermissions))
     }
 
-    @SuppressWarnings("PrintStackTrace")
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
     fun applyStoringPermissions(miniAppCustomPermission: MiniAppCustomPermission) {
-        try {
-            val jsonToStore: String = Gson().toJson(orderByDefaultList(miniAppCustomPermission))
-            prefs.edit().putString(miniAppCustomPermission.miniAppId, jsonToStore).apply()
-        } catch (e: Exception) {
```

Comment:
Is there any reason to remove this exception?

Reply:
Actually, why there is the try catch exception here?

Reply:
I think, it's good to have an exception catchup as there could be a case as documentation. It's not obvious but code quality matters.

<img width="670" alt="Screen Shot 2020-12-03 at 18 28 23" src="https://user-images.githubusercontent.com/66930373/100990639-75f01680-3595-11eb-8616-53d9fbb3ecfe.png">

Reply:
I think if there is error on json parsing, it should be addressed because there must be no error here.